draft-howard-rats-coserv.md

```diff
@@ -104,6 +105,37 @@ The terminology from CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {{-cose}} applies;
 in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
 and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.
 
+# Aggregation and Trust Models {#secaggregation}
+
+The roles of Endorser or Reference Value Provider might sometimes be fulfilled by aggregators, which collect from multiple supply chain sources, or even from other aggregators, in order to project a holistic view of the endorsed system.
+The notion of such an aggregator is not explicit in the RATS architecture.
+In practice, however, supply chains are complex and multi-layered.
+Supply chain sources can include silicon manufacturers, device manufacturers, firmware houses, system integrators, service providers and more.
+In practical terms, an Attester is likely to be a composite entity, formed of components from across such supply chains.
+Evidence would be likewise structured, with contributions from different segments of the Attester's overall anatomy.
+A Verifier for such Evidence may find it convenient to contact an aggregator as a single source of truth for endorsements.
```

Comment:
We seem to use "Endorsement and Reference Values” consistently throughout the document.
Using the term “endorsements" here may cause confusion.
I suggest we do one of two things:
1. Replace "endorsements" with "Endorsement and Reference Values", or
2. Introduce lowercase “endorsements" as shorthand for "Endorsement and Reference Values” in the terminology section, and then perform a mass substitution.